test/test_evaluation/test_abstract_evaluator.py

```diff
@@ -92,46 +90,6 @@ def test_finish_up_model_predicts_NaN(self):
             {"error": "Model predictions for optimization set " "contains NaNs."},
         )
 
-        # NaNs in prediction validation
-        predictions_ensemble[5, 2] = 0.5
-        predictions_valid[5, 2] = np.NaN
-        _, loss, _, additional_run_info = ae.finish_up(
-            loss=0.1,
-            train_loss=0.1,
-            opt_pred=predictions_ensemble,
-            valid_pred=predictions_valid,
-            test_pred=predictions_test,
-            additional_run_info=None,
-            final_call=True,
-            file_output=True,
-            status=StatusType.SUCCESS,
-        )
-        self.assertEqual(loss, 1.0)
-        self.assertEqual(
-            additional_run_info,
-            {"error": "Model predictions for validation set " "contains NaNs."},
-        )
-
-        # NaNs in prediction test
```

Comment:
We could keep the testing of the `test` set, right?

Reply:
The one above this comment is for validation but sure, the one below I could just remove the `predictions_valid` part and put it back in